v2/astradb-to-bigquery/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <artifactId>dynamic-templates</artifactId>
+        <groupId>com.google.cloud.teleport.v2</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>astradb-to-bigquery</artifactId>
+    <name>AstraDB To BigQuery</name>
+    <description>Copy a Table hosted in Cassandra on DataStax Astra to BigQuery</description>
+
+    <properties>
+        <astra-io.version>0.2.3</astra-io.version>
+        <astra-sdk.version>0.5</astra-sdk.version>
+    </properties>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>com.datastax.astra</groupId>
+            <artifactId>beam-sdks-java-io-astra</artifactId>
+            <version>${astra-io.version}</version>
```

Comment:
Curious: Any reason to not contribute to Beam? I think it would be a great addition to Beam.

Reply:
No particular reason on my end. Code is OSS and available here https://github.com/clun/beam-sdks-java-io-astra. 
I was not sure such an SDK (for an enterprise product) would have been accepted.

Reply:
Now that this component has been upgraded to driver4x it is make even more sense as driver3x will not get new features development.